Makefile

```diff
@@ -103,6 +103,12 @@ Q := $(if $V,,@)
 	$Q ln -s ../bin .GOPATH/bin
 	$Q touch $@
 
+.GOPATH/.setup:
+	if [ -a ./bin/gvt ]; \
```

Comment:
The bin folder is .gitignored, it's not on the machine of anyone who didn't run setup themselves.

Use `grep "/.GOPATH" .gitignore` (see above).